qiskit_optimization/algorithms/cplex_optimizer.py

```diff
@@ -61,6 +70,7 @@ def __init__(self, disp: bool = False) -> None:
             )
 
         self._disp = disp
+        self._cplex_parameters = {} if cplex_parameters is None else cplex_parameters
```

Comment:
By the way, this can be expressed by `self._cplex_parameters = cplex_parameters or {}`